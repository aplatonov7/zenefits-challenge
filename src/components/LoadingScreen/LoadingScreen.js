import React from 'react'

import './LoadingScreen.css'

const LoadingScreen = () => (
  <div className="LoadingScreen__container">
    <div className="loader" />
  </div>
)

export default LoadingScreen