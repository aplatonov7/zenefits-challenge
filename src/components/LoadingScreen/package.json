{
  "name": "LoadingScreen",
  "version": "0.0.0",
  "private": true,
  "main": "./LoadingScreen.js"
}