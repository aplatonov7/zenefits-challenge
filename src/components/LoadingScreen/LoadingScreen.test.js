import React from 'react'

import LoadingScreen from './LoadingScreen'

describe('LoadingScreen component', () => {
  it('Should match snapshot', () => {
    expect(renderer.create(<LoadingScreen />).toJSON()).toMatchSnapshot()
  })
})