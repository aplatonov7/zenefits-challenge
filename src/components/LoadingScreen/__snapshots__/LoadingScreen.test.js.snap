// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingScreen component Should match snapshot 1`] = `
<div
  className="LoadingScreen__container"
>
  <div
    className="loader"
  />
</div>
`;
